src/request.ts

```diff
@@ -580,6 +591,15 @@ class DatastoreRequest {
    *     [here](https://cloud.google.com/datastore/docs/articles/balancing-strong-and-eventual-consistency-with-google-cloud-datastore).
    * @param {object} [options.gaxOptions] Request configuration options, outlined
    *     here: https://googleapis.github.io/gax-nodejs/global.html#CallOptions.
+   * @param {object} [options.integerTypeCastOptions] Configurations to
+   *     optionally wrap `integerValue` in Datastore Int object and optionally
+   *     provide an `integerTypeCastFunction` to handle `integerValue` conversion
+   * @param {boolean} [options.integerTypeCastOptions.wrapnumbers=false]
```

Comment:
Could we promote or alias `wrapNumbers` one level up on the `options` object (`options.wrapNumbers`)? Using this should opt the user out of the custom functionality of `integerTypeCastFunction`, as it's the simplest option to have number safety.

Reply:
I think I would like to wait to see if that is necessary before cluttering things up?

@bcoe

Reply:
We talked about this in the chat, @AVaksman was going to make it work. I don't see the clutter though-- focusing on ease of use for the user, "options.wrapNumbers" is easier to remember and more intuitive than "options.integerTypeCastOptions = { wrapNumbers: true }".